R/functions-Chromatograms.R

```diff
@@ -0,0 +1,104 @@
+## functions for Chromatograms class
+
+.validChromatograms <- function(x) {
+    msg <- character()
```

Comment:
Ok, I was not aware of a new coding style (do you have a link to Martin Morgan's coding style?). But I really like the idea to use as much `base` functions as possible. So go ahead. Let's forget `Biobase::validMsg`.